test/configs/zombie_tanssi_warp_sync.json

```diff
@@ -72,7 +72,9 @@
                 {
                     "name": "Collator1000-03",
                     "command": "../target/release/tanssi-node",
-                    "args": ["--no-hardware-benchmarks", "--database=paritydb", "--wasmtime-precompiled=wasm"]
+                    "args": ["--no-hardware-benchmarks", "--database=paritydb", "--wasmtime-precompiled=wasm", "-ldebug,sync=trace",
+                        "-- --rpc-port 9952 --rpc-cors all --unsafe-rpc-external --rpc-methods unsafe"
```

Comment:
why this change? I mean here you are nowhere indicating that you want to skip the full download is that right?

Reply:
This is to enable testing:
* `-lsync=trace` is to check the gap sync log
* rpc is to enable collator rpc, which would otherwise be on a random port

Reply:
ok cool